bql/builtin.go

```diff
@@ -133,10 +170,30 @@ func createFileSource(ctx *core.Context, ioParams *IOParams, params data.Map) (c
 		}
 	}
 
+	var repeat int64
+	if v, ok := params["repeat"]; ok {
+		r, err := data.AsInt(v)
+		if err != nil {
+			return nil, fmt.Errorf("'repeat' parameter must be an integer: %v", err)
+		}
+		repeat = r
+	}
+
+	var interval time.Duration
+	if v, ok := params["interval"]; ok {
```

Comment:
I thought about it and found out that a negative interval has the same meaning as 0 because the source checks with `if s.interval > 0 {` whether the interval is specified or not. So, just documenting the specification as "when the interval is less than or equal to 0, the source emits tuples as fast as possible" may be fine. What do you think?